packages/web/src/scss/components/Header/_Header.scss

```diff
@@ -18,229 +17,116 @@
     background-color: transparent;
 }
 
-.Header__mobileOnlyActions {
+.Header__actions {
     display: flex;
-    gap: theme.$mobile-only-actions-gap;
+    flex: none;
+    gap: theme.$header-actions-gap-mobile;
     align-items: center;
-    margin-left: auto;
 }
 
-.Header__bar,
-.Header__dialog {
-    $_shadow-offset: -2rem;
-
-    position: fixed;
-    top: 0;
-    right: 0;
-    bottom: 0;
-    z-index: 1;
-    display: grid;
-    width: 280px;
-    color: theme.$pane-color;
-    background-color: theme.$pane-background-color;
-    visibility: hidden;
-    transform: translateX(100%);
+.Header__actions--mobileOnly {
+    margin-inline-start: auto;
+}
 
-    &::after {
-        @include tools.create-cover($top: $_shadow-offset, $bottom: $_shadow-offset);
+.Header__actions--desktopOnly {
+    display: none;
+}
 
-        content: '';
-        z-index: -1;
-        box-shadow: theme.$pane-shadow;
+@include breakpoint.up(theme.$header-breakpoint) {
+    .Header__actions {
+        gap: theme.$header-actions-gap-desktop;
+        align-self: stretch;
     }
 
-    &::backdrop {
-        background-color: theme.$backdrop-background-color;
+    .Header__actions--mobileOnly {
+        display: none;
     }
 
-    @media (prefers-reduced-motion: no-preference) {
-        transition-property: visibility, transform;
-        transition-duration: 0.25s;
-        transition-timing-function: ease-in-out;
+    .Header__actions--desktopOnly {
+        display: flex;
     }
-}
 
-.Header__content {
-    display: flex;
-    flex-direction: column;
-    overflow-y: auto;
-    -webkit-overflow-scrolling: touch;
-}
-
-.Header__actions {
-    display: flex;
-    flex-direction: column;
-    gap: theme.$actions-row-gap;
-    padding: theme.$actions-padding-y theme.$actions-padding-x;
-}
-
-.Header__actions--primary {
-    flex: none;
-}
-
-.Header__actions--secondary {
-    flex: 1;
+    .Header__actions--secondary {
+        margin-inline-start: auto;
+    }
 }
 
 .Header__nav {
-    margin-inline: -1 * theme.$actions-padding-x;
+    display: flex;
+    align-self: stretch;
     list-style: none;
 }
 
 .Header__navItem {
     display: flex;
 }
 
-// stylelint-disable selector-no-qualifying-type -- Style links with the semantics of a button.
-.Header__close,
-button.Header__link {
+// stylelint-disable-next-line selector-no-qualifying-type -- Safely reset links with the semantics of a button using `:where()`.
```

Comment:
do we have to mention that we use `:where()` for a safe reset?

Reply:
I was hesitating with this, `:where()` is still a new thing… I'll delete it though, let's be optimistic.